eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1142,16 +1142,22 @@ class Channel(val nodeParams: NodeParams, wallet: EclairWallet, remoteNodeId: Pu
       wallet.rollback(fundingTx)
       stay
 
-    case Event(INPUT_DISCONNECTED, _) => stay // we are disconnected, but it doesn't matter anymoer
+    case Event(INPUT_DISCONNECTED, _) => stay // we are disconnected, but it doesn't matter anymore
   })
 
   when(OFFLINE)(handleExceptions {
     case Event(INPUT_RECONNECTED(r), d: HasCommitments) =>
       forwarder ! r
+
+      val yourLastPerCommitmentSecret = d.commitments.remotePerCommitmentSecrets.lastIndex.flatMap(d.commitments.remotePerCommitmentSecrets.getHash).getOrElse(Sphinx.zeroes(32))
```

Comment:
Where did you get the default value `Sphinx.zeroes(32)`? Can't find it in the spec

Reply:
nitpick: `yourLastPerCommitmentSecret ` should be of type `Scalar`

Reply:
I'm sure it was there in spec but can't find it either. What should be there if `lastIndex` is `None`?

Reply:
if `lastIndex`=`None` then that's our first commitment, and we shouldn't send anything maybe?